scripts/getnomad.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-NOMAD_VERSION=1.3.1
+NOMAD_VERSION=1.4.0-beta.1
```

Comment:
We should update this after GA, but at least this PR gives a starting point for any work that needs to be done before that like #286 and #284.